lib/forwardanalyzer.cpp

```diff
@@ -51,15 +51,15 @@ struct ForwardTraversal {
     enum class Progress { Continue, Break, Skip };
     enum class Terminate { None, Bail, Escape, Modified, Inconclusive, Conditional };
     ForwardTraversal(const ValuePtr<Analyzer>& analyzer, const Settings* settings)
-        : analyzer(analyzer), settings(settings), actions(Analyzer::Action::None), analyzeOnly(false), analyzeTerminate(false)
+        : analyzer(analyzer), settings(settings)
     {}
     ValuePtr<Analyzer> analyzer;
     const Settings* settings;
-    Analyzer::Action actions;
-    bool analyzeOnly;
-    bool analyzeTerminate;
+    Analyzer::Action actions{ Analyzer::Action::None };
+    bool analyzeOnly{};
+    bool analyzeTerminate{};
```

Comment:
These should be initialized as `= false`. Using `{}` will not always zero initialize the memory.

Reply:
Huh? Do you mean the `bool` members?

Reply:
yes

Reply:
But `bool b{};` will always be `false`.